appcues/src/main/java/com/appcues/action/ActionProcessor.kt

```diff
@@ -44,6 +44,11 @@ internal class ActionProcessor(override val scope: AppcuesScope) : AppcuesCompon
         transformQueue(actions).toMutableList().forEach { it.execute() }
     }
 
+    // Calls Process on a different Job to unlock existing thread
```

Comment:
and a comment here to say this is the version used by post flow completion actions - and update the comment on L34-35 please to remove the reference to post flow completion actions, as that one is only used by the navigation actions now.